examples/stress-mock/app.js

```diff
@@ -22,9 +22,9 @@ var runAsync = async () => {
     }
     fs.writeFileSync("./generated/varkes_config.json", JSON.stringify(config))
     try {
-        app.use(await connectorApp("./generated/varkes_config.json"))
-        app.use(await odataApp.init("./generated/varkes_config.json"))
-        app.use(await openapiApp("./generated/varkes_config.json"))
+        app.use(await connectorApp("./generated/varkes_config.json", null, __dirname))

```

Comment:
Please change the order of the arguments so that a null value is not needed.
That nodePort param will be removed very soon as it is just a workaround